zebrad/src/application.rs

```diff
@@ -89,7 +89,11 @@ impl Application for ZebradApp {
         let terminal = Terminal::new(self.term_colors(command));
         // This MUST happen after `Terminal::new` to ensure our preferred panic
         // handler is the last one installed
-        color_eyre::install().unwrap();
+        color_eyre::config::HookBuilder::default()
+            .issue_url("https://github.com/ZcashFoundation/zebra/issues/new")
+            .add_issue_metadata("version", "3.0.0-alpha.0")
```

Comment:
I haven't done it yet but I want to consider whether we should add metadata about the system as well to these issues. Here's the example this feature was based on where the dev also added the `os_type` and `os_release` to their issue template.

https://twitter.com/chrisbiscardi/status/1306341323348144128

Reply:
:+1: seems like a good idea, but also something that we can add incrementally in future PRs?

Reply:
:100: